tests/plugins/kernel_install/utils_test.sh

```diff
@@ -52,37 +62,46 @@ function test_ask_yN()
 
 function test_human_list_installed_kernels()
 {
-  local count=0
-
   declare -a expected_out=(
-    "" # Extra espace in the beginning
     "5.5.0-rc2-VKMS+"
     "5.6.0-rc2-AMDGPU+"
     "linux"
   )
 
+  printf "%s\n" "${expected_out[@]}" > "$REMOTE_KW_DEPLOY/MANAGED_KERNELS"
+
```

Comment:
Something is wrong here. When I tried to run all tests locally, I got:
```
Running test [utils_test]
=========================================================
test_cmd_manager
test_ask_yN
test_human_list_installed_kernels
[sudo] password for siqueira:
```

Idk why our CI did not complain. Any idea? I suspect of `sudo grep -q "$kernel" "$managed_kernels_path"`

Reply:
I think the CI runs a passwordless sudo, so there is no problem.

Either way, how to fix this? We need permissions to access the /root/kw_deploy folder

Reply:
Could you check the CI? Something is wrong; see my comment in the deploy parser refactor.

Btw, we can mock the `sudo` command in that test. Or you can use the `cmd_manager` to validate the command.